### PR TITLE
Deploy using grunt

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Removes tracking of styleguide from git repo
- Adds grunt function `grunt deploy` 
- `grunt deploy` compiles, packages, and pushes the style guide to the github.io repository.

# Needed By (Date)
- Whenever

# Urgency
- Real low

# Steps to Test

1. Check out this branch
2. Run npm install
3. Run `grunt deploy`
4. Visit https://github.com/SU-SWS/decanter.github.io and review the last commit for a recent timestamp.
5. Wait a few minutes for the cache to clear and review http://decanter.stanford.edu

# Affected Projects or Products
- Decanter website.

# Associated Issues and/or People
-  ? Probably but I haven't looked. ?

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by josephgknox** [comment]
I merged in `master` to address conflicts after merging the footer work in.

**Comment by josephgknox** [comment]
Woot! No more tracking of a million style guide related file changes.

It seems that `grunt deploy` updates decanter.stanford.edu, even if you're on a feature branch. Is that right? Is there a danger in running that before code has been reviewed?

**Comment by sherakama** [comment]
Yeah, we will have to talk about practice on this and how we want to tag+release these builds but this is a start.

File: core/templates/components/atoms/media-block-img/media-block-img.twig [hunk 1]
```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (josephgknox) [reply core/templates/components/atoms/media-block-img/media-block-img.twig]: Right on.

**Pull request merged**